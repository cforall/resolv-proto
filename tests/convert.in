f-u2s1 0 3
f-s1s1 3 3
f-s0s2 2 4
f-s0s1 2 3
%%
f( 2 2 )
