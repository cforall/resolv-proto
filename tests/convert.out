[Void / (0,0,0,0,1)] f-s0s1( 2 2 => 3 )
